### PR TITLE
(feat) O3-3599 : Update tutorial modal to O3 modal system

### PR DESCRIPTION
**Pull request by Vijaykv5**
## Requirements
- [x] This PR has a title that briefly describes the work done including a [conventional commit](https://o3-dev.docs.openmrs.org/#/getting_started/contributing?id=your-pr-title-should-indicate-the-type-of-change-it-is) type prefix and a Jira ticket number if applicable. See existing PR titles for inspiration.

#### For changes to apps
- [x]  My work conforms to the [**OpenMRS 3.0 Styleguide**](https://om.rs/styleguide) and [**design documentation**](https://zeroheight.com/23a080e38/p/880723-introduction).

#### If applicable
- [ ] My work includes tests or is validated by existing tests.

## Screenshots

https://github.com/user-attachments/assets/ea65980b-b50e-490c-9f14-b643190a9f1a

## Related Issue

[O3-3599](https://openmrs.atlassian.net/browse/O3-3599)

## Other

[O3-3599]: https://openmrs.atlassian.net/browse/O3-3599?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/index.ts [hunk 1]
```diff
@@ -18,3 +18,5 @@ export function startupApp() {
 export const root = getAsyncLifecycle(() => import('./root.component'), options);
 
 export const tutorial = getSyncLifecycle(Tutorial, options);
+
+export const tutorialmodal = getAsyncLifecycle(() => import('./tutorial/modal.component'), options);
```
Comment (Piumal1999) [inline src/index.ts:22]:
```suggestion
export const tutorialModal = getAsyncLifecycle(() => import('./tutorial/modal.component'), options);
```

**Comment by Piumal1999** [comment]
Good job @Vijaykv5. Could you please resolve the conflicts?

File: src/index.ts (hunk 1 above)
Comment (Vijaykv5) [reply src/index.ts]: > 

Done :)

**Review by Piumal1999 — APPROVED** [review]
Good job @Vijaykv5

**Pull request merged**